docs/source/programming_resources/shared/configuring_android/Configuring-Your-Android-Devices.rst

```diff
@@ -0,0 +1,545 @@
+Configuring your Android Devices
+================================
+
+What Needs to Be Configured for My Control System?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Control Hub Configuration 
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Teams who are using a Control Hub with the integrated Robot Controller
+will only need to configure a single smartphone for use as a Driver
+Station. The process is as follows: 
+
+*  Rename the smartphone to "<TEAM NUMBER>-DS" (where <TEAM NUMBER> is replaced by your team number). 
+*  Install the Driver Station (DS) app onto the Driver Station phone. (The DS app is pre-installed on the REV Driver Hub.) 
+*  Put your phone into Airplane Mode (with the WiFi radio still on). 
+*  Pair (i.e., wirelessly connect) the Driver Station to the Control Hub.
+
+.. image:: images/ControlHubAndPhone.jpg
+   :align: center
+
+|
+
+References to the Driver Station smartphone may instead apply to the
+`REV Driver
+Hub <https://docs.revrobotics.com/rev-control-system/driver-hub-gs>`__,
+which is preloaded with the FTC Driver Station (DS) app.
+
+**IMPORTANT NOTE:** Eventually the Control Hub will need be renamed so
```

Comment:
Use `.. note::` or `.. warning::`

Reply:
I'd advocate for consistent casing.

Reply:
Yeah, I guess that begs the question of whether or not we're editing and revising all of Chris's old documentation or simply trying to convert it over.

Reply:
done